ddl/backfilling.go

```diff
@@ -1095,14 +1106,15 @@ func (dc *ddlCtx) writePhysicalTableRecord(sessPool *sessionPool, t table.Physic
 		if err != nil {
 			return errors.Trace(err)
 		}
-
-		if len(remains) == 0 {
-			if ingestBeCtx != nil {
-				ingestBeCtx.EngMgr.ResetWorkers(ingestBeCtx, job.ID, reorgInfo.currElement.ID)
-			}
-			break
+		if len(remains) > 0 {
+			startKey = remains[0].StartKey
+		} else {
+			lastFinishedKey := kvRanges[len(kvRanges)-1].EndKey
+			startKey = lastFinishedKey.Next()
```

Comment:
Why use the Next()? The end key is not covered yet.

Reply:
Updated.